src/main/java/de/mediathekview/mserver/crawler/br/BrWebAccessExecution.java

```diff
@@ -1,21 +1,21 @@
-/*
- * BrWebAccessExecution.java
- * 
- * Projekt    : MServer
- * erstellt am: 07.10.2017
- * Autor      : Sascha
- * 
- * (c) 2017 by Sascha Wiegandt
- */
-package de.mediathekview.mserver.crawler.br;
-
-import java.net.MalformedURLException;
-
-import com.google.gson.JsonSyntaxException;
-
-@FunctionalInterface
-public interface BrWebAccessExecution {
-    void run() throws JsonSyntaxException,
-                      IllegalArgumentException,
-                      MalformedURLException;
-}
+/*
+ * BrWebAccessExecution.java
+ * 
+ * Projekt    : MServer
+ * erstellt am: 07.10.2017
+ * Autor      : Sascha
+ * 
+ * (c) 2017 by Sascha Wiegandt
+ */
+package de.mediathekview.mserver.crawler.br;
+
+import java.net.MalformedURLException;
+
+import com.google.gson.JsonSyntaxException;
+
+@FunctionalInterface
+public interface BrWebAccessExecution {
+    void run() throws JsonSyntaxException,
+                      IllegalArgumentException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)